galaxy/Dockerfile

```diff
@@ -199,4 +203,4 @@ EXPOSE :9002
 VOLUME ["/export/", "/data/", "/var/lib/docker"]
 
 # Autostart script that is invoked during container start
-CMD ["/usr/bin/startup"]
+CMD ["/usr/bin/startCondor.sh"]
```

Comment:
In hindsight, this will indeed break all of the `-e magic`. That's OK for testing, but egads don't merge this!